transcom/mymove
# B 18735 int auto check the move management box

## [B-18735](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A906270)

## Summary

When approving a shipment as a TOO the Move Management checkbox should be automatically checked by default and cannot be unchecked. This will ensure Move Management service will be added to each move. 

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

## Test without Counseling Fee

1. Login as a Customer and create a new HHG shipment 
2. Submit the shipment
3. Login to Office app as a Service Counselor and approve the HHG shipment
4. Login to Office app as a TOO
5. Select the HHG shipment
6. Verify the Move Management checkbox is selected and disabled
7. Check the HHG checkbox above the Move Management checkbox and verify the Approve button is enabled
8. Click the Approve button

![image](https://github.com/transcom/mymove/assets/149000222/4811f501-1fef-4618-8636-b20c3d72491b)


## Test with Counseling Fee

1. Login as a Customer and create a new HHG shipment using Mayport, FL, 32233 as the pickup and destination and duty locations to ensure the shipment will be sent for GHC Counseling
2. Submit the shipment
4. Login to Office app as a TOO using CNNQ for GBLOC
5. Select the HHG shipment
6. Verify the Move Management checkbox is selected and disabled
7. Optionally check the box for Counseling
8. Check the HHG checkbox above the Move Management checkbox and verify the Approve button is enabled
9. Click the Approve button
![image](https://github.com/transcom/mymove/assets/149000222/fd81c40e-94fc-42f9-9624-cf1479bf3d93)

### Frontend


## Screenshots
